regimes/pt/pt.go

```diff
@@ -139,6 +166,36 @@ func New() *tax.Regime {
 								Percent: num.MakePercentage(60, 3),
 							},
 						},
+						Meta: cbc.Meta{
+							KeyTaxCode: "RED",
+						},
+					},
+					{
+						Key: TaxRateExempt,
+						Name: i18n.String{
+							i18n.EN: "Exempt",
+							i18n.PT: "Isenta",
+						},
+						Values: []*tax.RateValue{
+							{
+								Since:   cal.NewDate(2011, 1, 1),
+								Percent: num.MakePercentage(0, 3),
+							},
+						},
+						Meta: cbc.Meta{
+							KeyTaxCode: "ISE",
+						},
+					},
+					{
+						Key: TaxRateOther,
+						Name: i18n.String{
+							i18n.EN: "Other",
+							i18n.PT: "Outros",
+						},
+						Values: []*tax.RateValue{},
```

Comment:
No specific value is associated to this tax rate (if must be given in the percent value explicitly), so I left an empty array here. Do you think it's ok?

Reply:
As discussed in a previous commit, we'll leave the provider deal with this.